netplugin/agent/docker_event.go

```diff
@@ -31,9 +32,20 @@ import (
 	"golang.org/x/net/context"
 )
 
+func (ag *Agent) lbServiceExist() bool {
+	lbCfg := &mastercfg.CfgServiceLBState{}
+	plugin := ag.Plugin()
+	lbCfg.StateDriver = plugin.StateDriver
+
+	if lbList, err := lbCfg.ReadAll(); err == nil {
+		return len(lbList) > 0
+	}
+	return false
```

Comment:
In the case of etcd/consul being temporarily unavailable or some other error besides `key not found`, this function will return false... is that okay?